Two-Factor/README.md

```diff
@@ -22,13 +22,14 @@ The ```AGDroid Two-Factor``` project demonstrates how to include OTP functionali
 
 **Note**: The server side configuration is not mandatory for testing OTP with Android.
 
-1. Follow directions to install [Shoot 'n Share backend](https://github.com/aerogear/aerogear-backend-cookbook/blob/master/Shoot/README.md)
-1. Open the [console shoot-realm credentials](http://localhost:8080/auth/admin/master/console/#/realms/shoot-realm/required-credentials) and add TOTP in `Required User Credentials`
-1. Open Shoot 'n Share backend http://localhost:8080/shoot/photos
+### Configuring a testing server
+
+1. Follow directions to install [OTP-Demo](https://github.com/aerogear/aerogear-backend-cookbook/blob/master/OTP-Demo/README.md)
```

Comment:
let me fix it in ios-cookbook too, i suspect @edewit did a copy/paste of my wrong url :))